streampipes-extensions/streampipes-connect-adapters-iiot/pom.xml

```diff
@@ -127,6 +127,12 @@
             <groupId>org.apache.plc4x</groupId>
             <artifactId>plc4j-driver-s7</artifactId>
             <scope>runtime</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.fasterxml.woodstox</groupId>
+                    <artifactId>woodstox-core</artifactId>
+                </exclusion>
```

Comment:
Does this exclusion have any impact on the S7 driver?

Reply:
Hopefully not, we have to validate it with a PLC as soon as it is merged. This is included due to some convergence errors